packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md

```diff
@@ -1,14 +1,16 @@
----
-title: Basic sign-in flow with the password factor
+--- 
+title: Basic sign-in flow with password-only 
```

Comment:
Factor is used to call out that it's a "factor" - which we have many at Okta. If this were changed at all, we'd just remove "factor".